default.nix

```diff
@@ -0,0 +1,22 @@
+{ sources ? import ./nix/sources.nix
+, pkgs ? import sources.nixpkgs {
+    overlays = [ (import sources.rust-overlay) ];
+  }
+, rust-overlay ? pkgs.rust-bin.stable.latest.default
+}:
+  with pkgs;
+  mkShell {
+    name = "build";
+    buildInputs = [
+        # cargo tooling
+        cargo-deny
+        cargo-watch
+        pkgs.rust-bin.nightly."2022-07-01".rustfmt
```

Comment:
What's up with this?

Reply:
Oh, I can get rid of that. We need nightly in `radicle-git` for formatting -- which I copied this from. I'll remove it here.